bedita-app/app_controller.php

```diff
@@ -495,7 +495,7 @@ protected function checkLogin() {
                     $msg .= ': ' . $extUserId;
                 }
                 // avoid userid in logs / use ID instead or partially hide userid
-                $this->eventWarn(sprintf('external login failed: %s******', substr($userid, 0, -3)));
+                $this->eventWarn(sprintf('external login failed: %s', Mask::email($extUserId)));
```

Comment:
I'd expect `Mask::email($userid)` in here... but myabe `$extUserId` already contains the email to mask

Reply:
`$userid` seems undefined in this context… 🤔 `$extUserId` looked like it was the right variable to use, but I may be mistaken.

Reply:
I'll take a look on the file...